### PR TITLE
trame-vtk maintenance

### LINKED ISSUE #4020: subplot cannot zoom in
### Describe the bug, what's wrong, and what you expected.

Hi when plotting subplots using pyvista 0.38 on Jupyter lab, i can only zoom in and out with my mouse scroller on the first plot [0,0]. I remember old version allows me to zoom in and out the plot where my cursor is at. 

I also found that after running the cell that has the plotter, i have to click on the output window to display the plot. If i wait too long kernel will die from it.. 

Junting

### Steps to reproduce the bug.

```python
Insert code here
```

### System Information

```shell
--------------------------------------------------------------------------------
  Date: Tue Feb 14 10:00:29 2023 Eastern Standard Time

                OS : Windows
            CPU(s) : 12
           Machine : AMD64
      Architecture : 64bit
               RAM : 31.8 GiB
       Environment : Jupyter
        GPU Vendor : Intel
      GPU Renderer : Intel(R) UHD Graphics
       GPU Version : 4.5.0 - Build 30.0.101.1404

  Python 3.9.16 (main, Jan 11 2023, 16:16:36) [MSC v.1916 64 bit (AMD64)]

           pyvista : 0.38.1
               vtk : 9.2.5
             numpy : 1.24.2
           imageio : 2.25.0
            scooby : 0.7.1
             pooch : v1.6.0
        matplotlib : 3.6.3
           IPython : 8.9.0
          colorcet : 3.0.1
           cmocean : 2.0
        ipyvtklink : 0.2.3
              tqdm : 4.64.1
            meshio : 5.3.4
        jupyterlab : 3.6.1
         pythreejs : 2.4.1
             trame : 2.2.6
      trame_client : 2.5.1
      trame_server : 2.8.1
         trame_vtk : 2.0.17
--------------------------------------------------------------------------------
```

### Screenshots

mouse_scrolling
plotter_hang

### PR DESCRIPTION
**Pull request by banesullivan**
Maintenance PR to leverage the latest features and bug fixes from trame-vtk.

This is waiting on https://github.com/Kitware/trame-vtk/pull/31

resolves #4020

### CODE REVIEW

File: pyvista/trame/jupyter.py [hunk 1]
```diff
@@ -130,6 +130,8 @@ def launch_server(server=None, port=None, host=None):
     if not pyvista.global_theme.trame.enable_vtk_warnings:
         vtk_logger = logging.getLogger("vtkmodules.web.render_window_serializer")
         vtk_logger.disabled = True
+        trame_vtk_logger = logging.getLogger("trame_vtk.modules.vtk.serializers.serialize")
+        trame_vtk_logger.disabled = True
```
Comment (psavery) [inline pyvista/trame/jupyter.py:134]: There are actually several loggers in the `trame_vtk.modules.vtk.serializers` directory now. The parent one in the `__init__.py` file is set to `CRITICAL` level by default (and there are currently no critical messages, so nothing is printed by default), unless the `TRAME_SERIALIZE_DEBUG` environment variable is set, in which case all messages will be printed.

I wonder if disabling the parent `trame_vtk.modules.vtk.serializers` will disable all of the children too?

Comment (banesullivan) [reply pyvista/trame/jupyter.py:134]: > I wonder if disabling the parent trame_vtk.modules.vtk.serializers will disable all of the children too?

I had initially tried this without luck... just tried again and it works now... something must have change in the last few commits?

Comment (psavery) [reply pyvista/trame/jupyter.py]: Yes, I stopped setting the log level on the children, which means that they default to the parent log level (which makes it easier for us to turn them all on or off). And the parent now defaults to `CRITICAL`, so messages are no longer printed by default.

Comment (banesullivan) [reply pyvista/trame/jupyter.py]: Right, so this isn't even needed, thanks!

**Review by tkoyama010 — APPROVED** [review]
Thank you for working on this.

**Pull request merged**